### PR TITLE
JENA-1806: Update examples

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: jena-tdb/src-examples/tdb/examples/ExTDB_Txn2.java [hunk 1]
```diff
@@ -18,66 +18,56 @@
...
+ */
+public class ExTDB_Txn2 {
+    public static void main(String...argv) {
+        String directory = "MyDatabases/DB1";
+        Dataset dataset = TDBFactory.createDataset(directory);
 
-        // Start WRITE transaction. 
-        //   It's possible to read from the datet inside the write transaction.
+        // Start WRITE transaction.
+        // It's possible to read from the datet inside the write transaction.
```
Comment (kinow) [inline jena-tdb/src-examples/tdb/examples/ExTDB_Txn2.java]: s/datet/dataset ?

**Comment by Claudenw** [comment]
These changes are in place.  I assume this is to correct issues before moving them to the jena-examples project.  Is that corect?

**Comment by afs** [comment]
The changes are orthogonal to whether to move or not.

There were comments on email and on JENA-1806 so let's discuss the details there.

The idea of moving the examples is good - needs some details and refinement next.

File: jena-tdb/src-examples/tdb/examples/ExTDB_Txn2.java (hunk 1 above)
Comment (afs) [reply jena-tdb/src-examples/tdb/examples/ExTDB_Txn2.java]: Done!

**Pull request merged**